lib/caracal/core/models/table_model.rb

```diff
@@ -99,8 +101,33 @@ def cell_style(models, options={})
             m.apply_styles(options)
           end  
         end
+
+        # This method allows a user to specify one or more
+        # rows that should be rendered with that
+        # <w:cant_split /> property
+        #
+        # For example
+        #
+        # docx.table data do |t|
+        #   t.cant_split r.rows[0...-1]
+        # end
+        #
+        # would cause all rows in the table to be rendered
+        # with the <w:cant_split /> property.
+        def cant_split(models, options={})
+          models = rows.include?(models) ? [models] : models
+          models.each do |r|
+            @table_rows_cant_split[rows.index(r)] = true
```

Comment:
A potential future refactoring direction to consider but probably not yet worth pursuing; Rows within a table are gaining more behavior and distinct properties with these changes.  Right now, these properties are stored in distinct arrays and have to be referenced by index.  If we encapsulated rows in their own models, we could associate these properties with the rows directly instead of having to utilize the indexed array access approach.

Reply:
I agree 100%. Will talk with Cindy about creating a chore for this.